spec/robots/dor_repo/ocr/start_ocr_spec.rb

```diff
@@ -39,9 +37,32 @@
   context 'when the object is not opened and is not possible to OCR' do
     let(:version_open) { false }
     let(:possible) { false }
+    let(:note) { 'No files available or invalid object for OCR' }
+    let(:workflow) { 'ocrWF' }
+    let(:status) { 'skipped' }
+    let(:workflow_client) { instance_double(Dor::Workflow::Client, workflow: workflow_response) }
+    let(:workflow_response) { instance_double(Dor::Workflow::Response::Workflow, incomplete_processes: [start_ocr, fetch_files, ocr_create, end_ocr]) }
+    # NOTE: this is just mocking a workflow response, it doesn't actually have to be fully accurate
+    # we just need to expect we will update the status of all steps except start-ocr
+    let(:start_ocr) { instance_double(Dor::Workflow::Response::Process, name: 'start-ocr') }
+    let(:fetch_files) { instance_double(Dor::Workflow::Response::Process, name: 'fetch-files') }
+    let(:ocr_create) { instance_double(Dor::Workflow::Response::Process, name: 'ocr-create') }
+    let(:end_ocr) { instance_double(Dor::Workflow::Response::Process, name: 'end-ocr') }
+
+    let(:return_status) { perform.status }
 
-    it 'raise an error' do
-      expect { perform }.to raise_error('No files available or invalid object for OCR')
+    before do
+      allow(LyberCore::WorkflowClientFactory).to receive(:build).and_return(workflow_client)
+      allow(workflow_client).to receive(:update_status)
+    end
+
+    it 'sets all steps (except self) in the workflow to waiting and returns status of skipped for self' do
```

Comment:
is "waiting" possibly a copy/paste typo?  my read is that the other steps are also set to skipped?

Reply:
typo, will fix before merge